morph.json

```diff
@@ -9,7 +9,7 @@
   },
   "dependencies": {
     "@opam/dune": "*",
-    "@opam/lwt": "^4.0.0",
+    "@opam/lwt": ">=4.0.0",
```

Comment:
I've been annoyed with wrong constraints on multiple occasions, how about a slightly more conservative one:
```
>=4.0.0 <=5.x.x
```

Reply:
Lwt major version bumps usually don't affect users. Recent breaking changes affect obscure and little-used APIs, and this will continue to be the case. The vast majority of the API is very stable, so `>= 4.0.0` is what I would recommend in the specific case of Lwt.